tasks.py

```diff
@@ -433,7 +433,10 @@ def nbshell(context):
 @task(help={"service": "Name of the service to shell into"})
 def cli(context, service="nautobot"):
     """Launch a bash shell inside the running Nautobot (or other) Docker container."""
-    docker_compose(context, f"exec {service} bash", pty=True)
+    context.nautobot.local = False
+    command = "bash"
```

Comment:
Hmm... This seems like a strange one. Is forcing `local` really necessary?

Reply:
If you're running with `local=True` then `invoke cli` is functionally equivalent to just calling `bash`. I suppose an alternative here could be to have the command error out if you're running with `local=True`?

Reply:
Nah it's fine. Just clarifying.